website/docs/r/sql_failover_group.html.markdown

```diff
@@ -71,7 +71,7 @@ The following arguments are supported:
 
 * `databases` - A list of database ids to add to the failover group
 
--> **NOTE:** The failover group will create a secondary database for each database listed in `databases`. If the secondary databases need to be managed through Terraform, they should be defined as resources and a dependency added to the failover group to ensure the secondary databases are created first.
+-> **NOTE:** The failover group will create a secondary database for each database listed in `databases`. If the secondary databases need to be managed through Terraform, they should be defined as resources and a dependency added to the failover group to ensure the secondary databases are created first. Please refer to the detailed example which can be found in [the `./examples/mssql/failover_group` directory within the Github Repository](https://github.com/terraform-providers/terraform-provider-azurerm/tree/master/examples/mssql/failover_group)
```

Comment:
this resource is `sql` but the example is `mssql`- does this want updating to be `sql` until the replacement resources are present?